From 31a4037e49281e6e9b5097c34988834f20ad0c01 Mon Sep 17 00:00:00 2001
From: Harrison Carter <harrison.carter@thecapablehub.org>
Date: Thu, 5 Mar 2026 10:04:53 +0000
Subject: [PATCH 1/1] parser: remove line directive

Yocto flags TMPDIR as being present in source files 
generated by the parser. By removing the line switch we 
don't generate #line directives in the source but it 
squelches the warnings.

Signed-off-by: Harrison Carter <harrison.carter@thecapablehub.org>
---
 src/backend/parser/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/backend/parser/Makefile b/src/backend/parser/Makefile
index 9f1c4022bb..4f509c42bf 100644
--- a/src/backend/parser/Makefile
+++ b/src/backend/parser/Makefile
@@ -52,7 +52,7 @@ include $(top_srcdir)/src/backend/common.mk
 gram.h: gram.c
 	touch $@
 
-gram.c: BISONFLAGS += -d
+gram.c: BISONFLAGS += -d -l
 gram.c: BISON_CHECK_CMD = $(PERL) $(srcdir)/check_keywords.pl $< $(top_srcdir)/src/include/parser/kwlist.h
 
 
-- 
2.43.0

